admin-dev/themes/default/template/controllers/addresses/helpers/form/form.tpl

```diff
@@ -41,7 +41,7 @@
 	{if $input.type == 'text_customer'}
 		{if isset($customer)}
 			<div class="col-lg-9">
-				<a class="btn btn-default" href="?tab=AdminCustomers&amp;id_customer={$customer->id|intval}&amp;viewcustomer&amp;token={$tokenCustomer}">
```

Comment:
@matks here's one of the examples regarding hardcoded url.

Reply:
With an update of LegacyUrlConverter, we should be able to handle this 👍